src/xcode/ENA/ENA/Source/Services/TicketValidation/TicketValidation.swift

```diff
@@ -203,6 +211,8 @@ final class TicketValidation: TicketValidating {
 					completion(result)
 				}
 			case .failure(let error):
+				Log.error("Failed to request document of validation decorator with error: \(error)", log: .ticketValidation)
```

Comment:
Are we sure the error does not contain any private information?

Reply:
I hope that there is common consent to not put any private data into errors.
To be safe I looked into the ServiceError and the custom errors. Didn't see anything private.